README.rst

```diff
@@ -943,6 +943,37 @@ To use point ``STATICFILES_STORAGE`` to ``openwisp_utils.storage.CompressStaticF
 
     STATICFILES_STORAGE = 'openwisp_utils.storage.CompressStaticFilesStorage'
 
+``openwisp_utils.admin_theme.email.send_email``
```

Comment:
this section is inserted under storage utilities which doesn't look right, let's make a new section called "Admin them utilities".